### PR TITLE
🚀(helm) add base helm chart

### PR DESCRIPTION
**Pull request by wilbrdt**
## Proposal

Add a base helm chart to quickly deploy Mork on Kubernetes.

### CODE REVIEW

File: src/helm/mork/templates/api/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+metadata:
+  name: {{ include "mork.fullname" . }}-api
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    app.kubernetes.io/component: api
+    {{- include "mork.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "mork.selectorLabels" . | nindent 6 }}
```
Comment (claudusd) [inline src/helm/mork/templates/api/deployment.yaml:12]: I will add label ``app.kubernetes.io/component: api`` for this deployment.

File: src/helm/mork/templates/service.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "mork.fullname" . }}
```
Comment (claudusd) [inline src/helm/mork/templates/service.yaml:5]: You could add the suffix ``-api`` to match with the deployment name.

File: src/helm/mork/templates/service.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  ports:
+    - name: "{{ .Values.service.port }}-tcp"
+      port: {{ .Values.service.port }}
+      targetPort: {{ .Values.service.port }}
+      protocol: TCP
+      {{- if (eq .Values.service.type "NodePort") }}
+      nodePort: {{ .Values.service.nodePort.http }}
+      {{- end }}
+  type: {{ .Values.service.type }}
+  selector: {{- include "mork.selectorLabels" . | nindent 4 }}
```
Comment (claudusd) [inline src/helm/mork/templates/service.yaml:17]: Add the selector for the label ``app.kubernetes.io/component: api ``

**Review by claudusd — COMMENTED** [review]
Just minor change about the service and ensure the service match with the deployment API.

**Review by lunika — APPROVED** [review]

**Review by claudusd — APPROVED** [review]

**Pull request merged**